dbms/src/Flash/Coprocessor/DAGExpressionAnalyzer.cpp

```diff
@@ -615,6 +641,57 @@ void DAGExpressionAnalyzer::buildAggGroupBy(
             aggregated_columns.emplace_back(name, actions->getSampleBlock().getByName(name).type);
         }
     }
+#ifndef NDEBUG
+    for (const auto & p : key_ref_agg_func)
+    {
+        LOG_TRACE(Logger::get(), "key_ref_agg_func optimization: {} ref {}", p.first, p.second);
```

Comment:
```suggestion
        LOG_DEBUG(Logger::get(), "key_ref_agg_func optimization: {} ref {}", p.first, p.second);
```

Reply:
only for debug unit test, so no need to change